src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/html.groovy

```diff
@@ -24,59 +24,36 @@ def l = namespace(lib.LayoutTagLib)
 def st = namespace("jelly:stapler")
 
 l.layout(title:_("Pipeline Syntax: Reference"), norefresh: true) {
-    st.include(page: 'sidepanel')
```

Comment:
Ah, [whitespace-ignoring diff](https://github.com/jenkinsci/workflow-cps-plugin/pull/18/files?w=1) makes it clearer that you are just deleting a block from this page. So maybe just `git checkout master -- src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/html.groovy` and do that deletion so that the PR diff shows it naturally.